### PR TITLE
PORTALS-2137 follow-up :  set up all scss components file, and only show top nav bar…

### PR DESCRIPTION
**Pull request by jay-hodgson**
… on a few supported routes

### CODE REVIEW

File: src/components/TopNavBar.tsx [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
+      }, [location])
+    
+    
     return(
-    <React.Fragment>
-        <nav className='top-nav'> 
-            <NavLink to={'/'}><img src={logo} style={{height:'32px', paddingRight: '20px'}}/></NavLink>
+    <>
+        {isVisible && <nav className='top-nav'> 
+            <NavLink to={'/'}><img src={logo} alt="Sage Bionetworks logo" style={{height:'32px', paddingRight: '20px'}}/></NavLink>
```
Comment (jay-hodgson) [inline src/components/TopNavBar.tsx:33]: add an alt

File: src/components/TopNavBar.tsx [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
             {links.map(el => 
-                <NavLink to={el.path} activeClassName='isSelected' className={`nav-link`} >{el.text}</NavLink>
+                <NavLink to={el.path} key={el.text} activeClassName='isSelected' className={`nav-link`} >{el.text}</NavLink>
```
Comment (jay-hodgson) [inline src/components/TopNavBar.tsx:35]: important to set the key to a unique value for every child when creating children like this

Comment (jay-hodgson) [reply src/components/TopNavBar.tsx:35]: I think this informs react whether or not to rerender a node when comparing to the virtual dom

File: src/style/_Register.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+// .RegisterAccount1...
```
Comment (jay-hodgson) [inline src/style/_Register.scss:1]: example scss file for the account registration pages, to be filled in later

**Review by yjyoo773 — APPROVED** [review]
Thank you for reviewing and doing a follow up! Looks good

**Pull request merged**